modules/distr-tracing-tempo-getting-the-tempostack-instance-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="getting-tempo-stack-logs_{context}"]
+= Getting the TempoStack instance logs
+
+You can get the logs from a TempoStack component as follows.
+
+.Procedure
+
+. Locate the component you want to get the logs for. You can do this by listing all the deployments on an specific namespace.
+
+. Identify the pods that belong to this component.
```

Comment:
Again, how would a user do this? Are there specific commands involved or something specific they're looking for to identify it?

Reply:
Yes,
I think the standard way of doing this is using:
oc get pods
to list all pods, the ones that belongs to the tempostack will be in the format 

`tempo-<tempo-stack-name>-<component>`